sdk/python/tests/integration/registration/test_universal_registry.py

```diff
@@ -1482,9 +1502,47 @@ def test_apply_permission_success(test_registry):
     )
 
     test_registry.delete_permission("read_permission", project)
-    assert_project_uuid(project, project_uuid, test_registry)
+    assert_project_uuid(
+        project, project_uuid, DecisionStrategy.UNANIMOUS, test_registry
+    )
     permissions = test_registry.list_permissions(project)
-    assert_project_uuid(project, project_uuid, test_registry)
+    assert_project_uuid(
+        project, project_uuid, DecisionStrategy.UNANIMOUS, test_registry
+    )
     assert len(permissions) == 0
 
     test_registry.teardown()
+
+
+@pytest.mark.integration
+@pytest.mark.parametrize("test_registry", all_fixtures)
+def test_set_decision_strategy_success(test_registry):
+    project = "project"
+
+    entity = Entity(
+        name="driver_car_id",
+        description="Car driver id",
+        tags={"team": "matchmaking"},
+    )
+
+    # Register Entity
+    test_registry.apply_entity(entity, project)
```

Comment:
do we really need to apply the entity before testing the decision strategy? 🤔

Reply:
@dmartinol 
In some of the registry types the repo is not initialized if an apply is not performed before calling get_decision_strategy (The behavior is different between registry types and also registry store types) this is something we might open a jira to make the behavior between types more consistent